Yoast/javascript
# Add Portuguese and Indonesian word form support

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds word form support for Portuguese.
* [yoastseo] Adds word form support for Indonesian.
* [Yoast SEO Premium] Adds word form support for Portuguese.
* [Yoast SEO Premium] Adds word form support for Indonesian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* In Portuguese Premium plugin, try using different forms of words in the keyphrase and text, title, meta description, or slag and check whether the different forms of words are recognized. Examples of words to check:
_"abandona", "abandoná", "abandonada", "abandonadas", "abandonado", "abandonados", "abandonam", "abandonando", "abandonar", "abandonara", "abandonaram", "abandonasse", "abandono", "abandonou"_
or
_"acadêmica", "acadêmicas", "acadêmico", "acadêmicos"_
or
_"acessório", "acessórios"_
* In Indonesian Premium plugin, try using different forms of words in the keyphrase nad text, title, meta description, or slag and check whether the different forms of words are recognized. Exmples of words to check:
_"administratif", "administratifnya"_
or
_"aktif", "aktifkan", "aktifnya"_
or
_"perbuatan-perbuatan", "membuat-buat", "terbuat", "perbuat", "perbuatan", "perbuatanmu", "perbuatannya"_

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-358
Fixes https://yoast.atlassian.net/browse/LIN-357
